packages/rich-text/src/plugins/CommandPalette/utils/fetchAssets.ts

```diff
@@ -0,0 +1,26 @@
+import { FieldExtensionSDK } from '@contentful/app-sdk';
+import { entityHelpers } from '@contentful/field-editor-shared';
+
+export async function fetchAssets(sdk: FieldExtensionSDK, query: string) {
+  const assets = await sdk.space.getAssets({ query });
+
+  return assets.items.map((asset) => {
+    const displayTitle = entityHelpers.getAssetTitle({
+      asset,
+      localeCode: sdk.field.locale,
+      defaultLocaleCode: sdk.locales.default,
+      defaultTitle: 'Untitled',
+    });
+
+    return {
+      contentTypeName: 'Asset',
+      displayTitle,
+      id: asset.sys.id,
+      entry: asset,
```

Comment:
I'd propose renaming this prop name either to `asset` or `entity` if the type is reused for entries and assets.

Naming it entry and assigning an asset as the value is a bit confusing.